app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.tsx

```diff
@@ -78,7 +75,7 @@ function InPersonLocationFullAddressEntryPostOfficeSearchStep({
       const selected = transformKeys(selectedLocation, snakeCase);
       setInProgress(true);
       try {
-        await request(LOCATIONS_URL, {
+        await request(locationsURL, {
```

Comment:
Why do we create a scoped variable instead of referencing the imported value directly?

```suggestion
        await request(LOCATIONS_URL, {
```